src/solver/optimisation/opt_gestion_second_membre_cas_lineaire.cpp

```diff
@@ -40,7 +41,8 @@ static void shortTermStorageLevelsRHS(
             const int clusterGlobalIndex = storage.clusterGlobalIndex;
             int cnt = CorrespondanceCntNativesCntOptim
                         .ShortTermStorageLevelConstraint[clusterGlobalIndex];
-            SecondMembre[cnt] = storage.series->inflows[hourInTheYear];
+            const auto* inflows = storage.series->inflows.series.getColumn(year);
+            SecondMembre[cnt] = inflows[hourInTheYear];
```

Comment:
You may use `getCoefficient` or similar for a single instruction, `getColumn` is meant to be called before a loop on timesteps, like below

Reply:
yes